From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chris Adams <chris.adams@jollamobile.com>
Date: Thu, 5 Feb 2015 20:32:52 +1000
Subject: [PATCH] Always install to /usr/lib never /usr/lib64

---
 common-project-config.pri | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/common-project-config.pri b/common-project-config.pri
index e149d90..68f82de 100644
--- a/common-project-config.pri
+++ b/common-project-config.pri
@@ -50,11 +50,7 @@ exists( meego-release ) {
     ARCH = $$system(uname -m)
 }
 
-contains( ARCH, x86_64 ) {
-    INSTALL_LIBDIR = $${INSTALL_PREFIX}/lib64
-} else {
-    INSTALL_LIBDIR = $${INSTALL_PREFIX}/lib
-}
+INSTALL_LIBDIR = $${INSTALL_PREFIX}/lib
 
 # default library directory can be overriden by defining LIBDIR when
 # running qmake
-- 
2.37.2

