From c4642e105499ab846080ca5a1c5bdd18d7f9c4d0 Mon Sep 17 00:00:00 2001
From: Chris Adams <chris.adams@jollamobile.com>
Date: Thu, 5 Feb 2015 20:34:01 +1000
Subject: [PATCH 4/6] Always force client auth via request body

---
 src/oauth2plugin.cpp | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/src/oauth2plugin.cpp b/src/oauth2plugin.cpp
index 290adf0..5775700 100644
--- a/src/oauth2plugin.cpp
+++ b/src/oauth2plugin.cpp
@@ -602,17 +602,17 @@ void OAuth2Plugin::sendOAuth2PostRequest(QUrl &postData,
     request.setRawHeader(CONTENT_TYPE, CONTENT_APP_URLENCODED);
 
     if (!d->m_oauth2Data.ClientSecret().isEmpty()) {
-        if (d->m_oauth2Data.ForceClientAuthViaRequestBody()) {
+//        if (d->m_oauth2Data.ForceClientAuthViaRequestBody()) {
             postData.addQueryItem(CLIENT_ID, d->m_oauth2Data.ClientId());
             postData.addQueryItem(CLIENT_SECRET, d->m_oauth2Data.ClientSecret());
-        } else {
-            QByteArray authorization =
-                QUrl::toPercentEncoding(d->m_oauth2Data.ClientId()) + ":" +
-                QUrl::toPercentEncoding(d->m_oauth2Data.ClientSecret());
-            QByteArray basicAuthorization =
-                QByteArray("Basic ") + authorization.toBase64();
-            request.setRawHeader("Authorization", basicAuthorization);
-        }
+//        } else {
+//            QByteArray authorization =
+//                QUrl::toPercentEncoding(d->m_oauth2Data.ClientId()) + ":" +
+//                QUrl::toPercentEncoding(d->m_oauth2Data.ClientSecret());
+//            QByteArray basicAuthorization =
+//                QByteArray("Basic ") + authorization.toBase64();
+//            request.setRawHeader("Authorization", basicAuthorization);
+//        }
     } else {
         postData.addQueryItem(CLIENT_ID, d->m_oauth2Data.ClientId());
     }
-- 
2.17.1

