From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chris Adams <chris.adams@jollamobile.com>
Date: Thu, 5 Feb 2015 20:34:01 +1000
Subject: [PATCH] Always force client auth via request body

---
 src/oauth2plugin.cpp | 13 ++-----------
 1 file changed, 2 insertions(+), 11 deletions(-)

diff --git a/src/oauth2plugin.cpp b/src/oauth2plugin.cpp
index be39263..3eb3fef 100644
--- a/src/oauth2plugin.cpp
+++ b/src/oauth2plugin.cpp
@@ -708,17 +708,8 @@ void OAuth2Plugin::sendOAuth2PostRequest(QUrl &postData,
     request.setRawHeader(CONTENT_TYPE, CONTENT_APP_URLENCODED);
 
     if (!d->m_oauth2Data.ClientSecret().isEmpty()) {
-        if (d->m_oauth2Data.ForceClientAuthViaRequestBody()) {
-            postData.addQueryItem(CLIENT_ID, d->m_oauth2Data.ClientId());
-            postData.addQueryItem(CLIENT_SECRET, d->m_oauth2Data.ClientSecret());
-        } else {
-            QByteArray authorization =
-                QUrl::toPercentEncoding(d->m_oauth2Data.ClientId()) + ":" +
-                QUrl::toPercentEncoding(d->m_oauth2Data.ClientSecret());
-            QByteArray basicAuthorization =
-                QByteArray("Basic ") + authorization.toBase64();
-            request.setRawHeader("Authorization", basicAuthorization);
-        }
+        postData.addQueryItem(CLIENT_ID, d->m_oauth2Data.ClientId());
+        postData.addQueryItem(CLIENT_SECRET, d->m_oauth2Data.ClientSecret());
     } else {
         postData.addQueryItem(CLIENT_ID, d->m_oauth2Data.ClientId());
     }
-- 
2.37.2

