--- a/src/oauth2plugin.cpp
+++ b/src/oauth2plugin.cpp
@@ -140,7 +140,9 @@ void OAuth2Plugin::sendOAuth2AuthRequest()
     if (!d->m_oauth2Data.Display().isEmpty()) {
         url.addQueryItem(DISPLAY, d->m_oauth2Data.Display());
     }
-    url.addQueryItem(QString("type"), d->m_mechanism);
+    if (!((d->m_oauth2Data.Host().contains(QLatin1String("dropbox.com")))
+          || (d->m_oauth2Data.Host().contains(QLatin1String("dropboxapi.com")))))
+        url.addQueryItem(QString("type"), d->m_mechanism);
     if (!d->m_oauth2Data.Scope().empty()) {
         QString separator = QLatin1String(" ");
